Support/cmp/Modules/FindEigen.cmake

```diff
@@ -16,6 +16,7 @@ endif()
 
 SET(EIGEN_INCLUDE_SEARCH_DIRS
   ${EIGEN_INSTALL}/include/eigen3
+  /usr/include/eigen3
```

Comment:
We have issues when putting these types of paths in. Sometimes the wrong path is picked up during configure. Is there a better way to do this?